client/disputes/filters/config.ts

```diff
@@ -0,0 +1,167 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { displayStatus } from 'disputes/strings';
+
+interface DisputesFilterEntryType {
+	label: string;
+	value: string;
+}
+
+export interface DisputesFilterType {
+	label: string;
+	param: string;
+	staticParams: string[];
+	showFilters: () => boolean;
+	filters: DisputesFilterEntryType[];
+	defaultValue?: string;
+}
+
+const disputesStatusOptions = Object.entries( displayStatus )
+	.map( ( [ status, label ] ) => {
+		return { label, value: status };
+	} )
+	.filter( function ( el ) {
+		return el != null;
+	} );
+
+export const filters: [ DisputesFilterType, DisputesFilterType ] = [
+	{
+		label: __( 'Dispute currency', 'woocommerce-payments' ),
+		param: 'store_currency_is',
+		staticParams: [
+			'paged',
+			'per_page',
+			'orderby',
+			'order',
+			'search',
+			'filter',
+			'status_is',
+			'status_is',
+			'date_before',
+			'date_after',
+			'date_between',
+		],
+		showFilters: () => false,
+		filters: [
+			{
+				label: __( 'All currencies', 'woocommerce-payments' ),
+				value: '---',
+			},
+			// Other values are getting injected later, taking values from store.
+		],
+		defaultValue: '---',
+	},
+	{
+		label: __( 'Show', 'woocommerce-payments' ),
+		param: 'filter',
+		staticParams: [
+			'paged',
+			'per_page',
+			'orderby',
+			'order',
+			'search',
+			'store_currency_is',
+		],
+		showFilters: () => true,
+		filters: [
+			{
+				label: __( 'All disputes', 'woocommerce-payments' ),
+				value: 'all',
+			},
+			{
+				label: __( 'Advanced filters', 'woocommerce-payments' ),
+				value: 'advanced',
+			},
+		],
+	},
+];
+
+/*eslint-disable max-len*/
+export const advancedFilters = {
+	/** translators: A sentence describing filters for Disputes. */
+	title: __( 'Disputes match {{select /}} filters', 'woocommerce-payments' ),
+	filters: {
+		date: {
+			labels: {
+				add: __( 'Date', 'woocommerce-payments' ),
```

Comment:
- We have two columns that contain dates, 'Disputed on' and 'Respond by'. I would use 'Disputed on' label for clarity (my understanding is that this is the `created` filter)

Reply:
Updated